packages/react-data-grid/src/utils/domUtils.tsx

```diff
@@ -30,3 +30,12 @@ export function isPositionStickySupported(): boolean {
   }
   return positionSticky;
 }
+
+const userAgent = navigator && navigator.userAgent;
+
+export const isIE = userAgent.indexOf('MSIE ') > -1 || userAgent.indexOf('Trident/') > -1;
+export const isEdge = !isIE && !!window.StyleMedia;
+
+export function isIEOrEdge(): boolean {
```

Comment:
Doesn't need to be a function
```suggestion
export const isIEOrEdge() = isIE || isEdge;
```